### DISCUSSION TITLE
How to configure multiple peers to point traffic to a peer

### DISCUSSION THREAD

**Discussion by luobobo896** [netbirdio/netbird#1221] [Ideas & Feature Requests]
How to configure multiple peers to point traffic to a peer, such as configuring WireGuard alone, any 0. 0. 0. 0 at the opposite end of 0 traffic accesses the network as a springboard exit

**Comment by mlsmaycon** [comment]
Hello @luobobo896, if I understood correctly, you want to route all traffic from other peers through a single peer. Currently, even known that NetBird can route network ranges with /8 and larger, it doesn't support routing 0.0.0.0/0 yet.

You can see the docs at https://docs.netbird.io/how-to/routing-traffic-to-private-networks to configure network routes.
